src/Schema/Events/MyCollection.ts

```diff
@@ -0,0 +1,104 @@
+import { ContextModule } from "../Values/ContextModule"
+import { OwnerType } from "../Values/OwnerType"
+import { ActionType } from "."
+
+/**
+ * Schema describing 'Add Collected Artwork' events
+ * @packageDocumentation
+ */
+
+/**
+ * A user taps "add artwork" to add an artwork to My Collection.
+ *
+ * This schema describes events sent to Segment from [[AddCollectedArtwork]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "addCollectedArtwork",
+ *    context_module: "myCollectionHome",
+ *    context_owner_type: "myCollection"
+ *  }
+ * ```
+ */
+export interface AddCollectedArtwork {
+  action: ActionType.addCollectedArtwork
+  context_module: ContextModule.myCollectionHome
+  context_owner_type: OwnerType.myCollection
+  context_owner_id?: string
```

Comment:
We usually collect `slug` as well as `id` when applicable, why not in this case?

Reply:
good q. i don't know if collections in 'My Collection' will even have slugs, but I assume they will have an id, which i left optional.

Reply:
could throw it in as `?` optional to be consistent

Reply:
Most records with an `id` should also have a slug, but not sure of the details for an individual user's collection. I do wonder if we want to know the `owner_id` and `owner_slug` of the collected artwork?

Reply:
i left details about the collected artwork off of this event because it's possible a user can tap 'add artwork' and then exit the flow before submitting, but i suppose in those instances, 'destination' info would be null, while for completed submissions, the destination info would be present.

Reply:
actually, i take that back. the event will fire before we know anything about the added artwork, so i think in this case, we won't need 'destination' info.

Reply:
(there should be a backend event recorded for a submitted artwork to My Collection)